sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -153,23 +154,25 @@ public virtual Task<Response<string>> GetDigitalTwinAsync(string digitalTwinId,
         /// </remarks>
         /// <param name="digitalTwinId">The Id of the digital twin.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
+        /// <param name="options">The optional parameters for this request.</param>
         /// <returns>The application/json digital twin and the http response <see cref="Response{T}"/>.</returns>
         /// <exception cref="RequestFailedException">
         /// The exception that captures the errors from the service. Check the <see cref="RequestFailedException.ErrorCode"/> and <see cref="RequestFailedException.Status"/> properties for more details.
         /// </exception>
         /// <exception cref="ArgumentNullException">
         /// The exception is thrown when <paramref name="digitalTwinId"/> is <c>null</c>.
         /// </exception>
-        public virtual Response<string> GetDigitalTwin(string digitalTwinId, CancellationToken cancellationToken = default)
+        public virtual Response<string> GetDigitalTwin(string digitalTwinId, DigitalTwinsGetByIdOptions options = null, CancellationToken cancellationToken = default)
```

Comment:
Consider changing this from `DigitalTwinsGetByIdOptions ` -> `DigitalTwinsGetOptions `